src/main/java/seedu/address/storage/JsonTaskStorage.java

```diff
@@ -63,25 +63,23 @@ public Path getTask() {
 
     /**
      * Save the task to the tasklist.
-     *
-     * @param taskList TaskList to be saved.
-     * @throws IOException If tasklist cannot be found.
+     * @param roomTasks
+     * @throws IOException
      */
-    public void saveTask(TaskList taskList) throws IOException {
-        saveTasks(taskList, task);
+    public void saveTask(RoomTasks roomTasks) throws IOException {
+        saveTasks(roomTasks, task);
     }
 
     /**
      * Save the task to the tasklist with the path.
```

Comment:
is there a line separating the description and the `@param` in the java docs?

Reply:
Added the line separator!